packages/block-library/src/image/edit.native.js

```diff
@@ -430,6 +434,50 @@ export class ImageEdit extends Component {
 		this.onSetSizeSlug( newValue );
 	}
 
+	onSetFeatured( mediaId ) {
+		const { closeSettingsBottomSheet } = this.props;
+		setFeaturedImage( mediaId );
+		closeSettingsBottomSheet();
+	}
+
+	getSetFeaturedButton( isFeaturedImage ) {
+		const { attributes, getStylesFromColorScheme } = this.props;
+
+		const setFeaturedButtonStyle = getStylesFromColorScheme(
+			styles.setFeaturedButton,
+			styles.setFeaturedButtonDark
+		);
+
+		const setFeaturedButton = () => (
+			<BottomSheet.Cell
+				label={ __( 'Remove as Featured Image ' ) }
+				labelStyle={ [
+					setFeaturedButtonStyle,
+					styles.removeFeaturedButton,
+				] }
+				onPress={ () => this.onSetFeatured( 0 ) }
+			/>
+		);
+
+		const removeFeaturedButton = () => (
+			<BottomSheet.Cell
+				label={ __( 'Set as Featured Image ' ) }
+				labelStyle={ setFeaturedButtonStyle }
+				onPress={ () => this.onSetFeatured( attributes.id ) }
+			/>
+		);
+
+		return (
+			<>
```

Comment:
Hmm, do we need the (React) fragment here? Looks like we're returning a single component (`PanelBody` in this case) so I wonder if there's a pattern you are following here that opts for using a fragment anyway? 🤔

Reply:
Ah, good point, these were only needed when the `androidOnly` conditional was in use here, I overlooked removing them. I've gone ahead to remove them now. 🙇‍♀️